File: app/src/main/java/net/gtr/framework/rx/DialogObserverHolder.java
package net.gtr.framework.rx;

import android.support.v4.app.FragmentManager;

/**
 * Created by caroline on 2018/4/24.
 */

public interface DialogObserverHolder extends ApplicationObserverResourceHolder{
    FragmentManager getSupportFragmentManager();

    void showDialog(Actions4SimpleDlg actions4SimpleDlg);
}

File: app/src/main/java/net/gtr/framework/rx/DialogObserverHolder.kt
package net.gtr.framework.rx

import android.support.v4.app.FragmentManager

/**
 * Created by caroline on 2018/4/24.
 */

interface DialogObserverHolder : ApplicationObserverResourceHolder {
    fun getSupportFragmentManager(): FragmentManager?
    fun showDialog(actions4SimpleDlg: Actions4SimpleDlg)
}
